apps/site/pages/en/about/index.mdx

```diff
@@ -77,3 +77,44 @@ by using our [`child_process.fork()`](https://nodejs.org/api/child_process.html)
 communicate with. Built upon that same interface is the [`cluster`](https://nodejs.org/api/cluster.html) module,
 which allows you to share sockets between processes to enable load balancing
 over your cores.
+
+## Official Node.js Resources
+
+To ensure authenticity and security when working with Node.js, always use official sources. Avoid trusting emails,
+binaries, or downloads from unofficial sources.
+
+### Official Node.js Domains
+
+For downloading Node.js binaries and accessing official documentation, use only these domains:
+
+- [nodejs.org](https://nodejs.org)
+- [nodejs.dev](https://nodejs.dev)
+- [iojs.org](https://iojs.org)
+
+### Official npm Packages
+
+The Node.js team maintains the following official npm package scopes:
+
+- `@node-core`
+- `@pkgjs`
+
+Additionally, the Node.js team maintains packages published by the `nodejs-foundation` npm account.
+
+Using packages from these scopes and accounts guarantees that you are working with officially supported Node.js components.
+
+### Official GitHub Organizations
+
+Node.js and related projects are maintained under these official GitHub organizations:
+
+- [nodejs](https://github.com/nodejs)
+- [pkgjs](https://github.com/pkgjs)
+
+### Official Communication Channels
+
+Node.js and the OpenJS Foundation communicate through various official and community-supported channels. You can find details on
+how to get involved on the [Get Involved](https://nodejs.org/en/about/get-involved) page.
+
+### Reporting Website Issues & Downtime
```

Comment:
Should we exaplain diff between website and api doc part ?

Reply:
Any strong opinion. Also if there are an issue we can still transfert it to right repo